tmt/utils.py

```diff
@@ -626,14 +628,39 @@ def _run(self,
         stdout_thread.start()
         if not join:
             stderr_thread.start()
+
+        # A bit of logging helpers for debugging duration behavior
+        start_timestamp = time.monotonic()
+
+        def _event_timestamp() -> str:
+            return f'{time.monotonic() - start_timestamp:.4}'
+
+        def log_event(msg: str) -> None:
+            self.debug(f'[command timestamps] {_event_timestamp()} {msg}', level=4)
```

Comment:
Find a better prefix - the goal is to be able to find these in debug log and verify cleaning up after a command did not take "too long"...